Build/CIRCLECI_SETUP.md

```diff
@@ -0,0 +1,246 @@
+# CircleCI variables setup
+
+This is how to make it build inside CircleCI. The licenses are needed to be
+added via environment variables inside the project settings within CircleCI.
+
+## Setup Unity licenses
+
+### Obtain Unity license .ulf files
+
+Repeat the following steps for the following versions:
+
+- 2018.4.14f1
+- 2019.2.11f1
+- 2020.1.0b6-linux-il2cpp
+
+1. Run `Build/local_docker_run.ps1` in PowerShell with `-UnityVersion` parameter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461394)